scripts/bootstrap/compile.sh

```diff
@@ -20,7 +20,7 @@ PROTO_FILES=$(ls src/main/protobuf/*.proto src/main/java/com/google/devtools/bui
 LIBRARY_JARS=$(find third_party -name '*.jar' | grep -Fv /javac-9-dev-r3297-4.jar | grep -Fv /javac-9-dev-4023-3.jar | grep -Fv /javac7.jar | grep -Fv JavaBuilder | grep -Fv third_party/guava | grep -Fv third_party/guava | grep -ve third_party/grpc/grpc.*jar | tr "\n" " ")
 GRPC_JAVA_VERSION=1.6.1
 GRPC_LIBRARY_JARS=$(find third_party/grpc -name '*.jar' | grep -e .*${GRPC_JAVA_VERSION}.*jar | tr "\n" " ")
-GUAVA_VERSIONE=23.1
+GUAVA_VERSION=23.1
```

Comment:
Is this even used somewhere?

Reply:
As far as I can tell, only the next line has a reference (to `GUAVA_VERSION`). I don't understand how the tests could pass previously.

Reply:
That means that it was matching everything before, we could just remove the whole logic probably. Anyway, your fix is correct, let's keep it that way for now

Reply:
What exactly was the intention of [e147372](https://github.com/bazelbuild/bazel/commit/e147372b073708631f1366bb2b9086ed413cb0c3)?